src/puptoo/process/profile.py

```diff
@@ -48,7 +48,7 @@
 from insights.parsers.systemctl_status_all import SystemctlStatusAll
 from insights.parsers.eap_json_reports import EAPJSONReports
 from insights.specs import Specs
-
+from insights.specs.datasources.pcp import ros_collect as ds_ros_collect
```

Comment:
@JoySnow -  Generally,`datasource` is for core collection, it's not a good idea to use it directly in plugins like puptoo. Please use the `insights.parsers.insights_client_conf.InsightsClientConf` instead.